Automattic/wp-calypso
# Remove/growth from comparison table

## Proposed Changes

* Remove Growth from comparison table. The product is not live until Dec 3rd

## Why are these changes being made?

* Growth is not being advertised front-facing yet

## Testing Instructions


1. Checkout this branch locally or use the Calypso Green live link
2. Go to `/features/comparison`
3. Make sure Growth is not showing
![image](https://github.com/user-attachments/assets/dd3ff6c0-1b35-40f4-988c-0debc9820156)


## Pre-merge Checklist
